app/src/main/java/com/asfoundation/wallet/repository/TransactionsHistoryPagingSource.kt

```diff
@@ -0,0 +1,43 @@
+package com.asfoundation.wallet.repository
+
+import androidx.paging.PagingSource
+import androidx.paging.PagingState
+import com.appcoins.wallet.core.network.backend.api.TransactionsApi
+import com.asfoundation.wallet.transactions.TransactionModel
+import com.asfoundation.wallet.transactions.toModel
+import retrofit2.HttpException
+import java.io.IOException
+
+
+class TransactionsHistoryPagingSource(
+  val backend: TransactionsApi,
```

Comment:
not sure if this is intentional, but this class could have the parameters injected in the constructor: 
class TransactionsHistoryPagingSource @Inject constructor(...)
Was it made this way to have a different PagingSource instance for each wallet?